packages/client/tsconfig.json

```diff
@@ -38,7 +38,7 @@
     "noImplicitReturns": true, /* Report error when not all code paths in function return a value. */
     "noFallthroughCasesInSwitch": true, /* Report errors for fallthrough cases in switch statement. */
     "noUncheckedIndexedAccess": true, /* Include 'undefined' in index signature results */
-    "noPropertyAccessFromIndexSignature": true, /* Require undeclared properties from index signatures to use element accesses. */
+    // "noPropertyAccessFromIndexSignature": true, /* Require undeclared properties from index signatures to use element accesses. */
```

Comment:
Suprising! why does the js client's tsconfig need to change for the new website? Can we revert?

Reply:
@orvn plz revert or explain here